### ISSUE TITLE
get_ligands

### ISSUE THREAD

**Issue by pauljmclaughlin** [williamgilpin/pypdb#53]
I find that "get_ligands" is not defined (running in Jupyter Notebooks- use "!pip install pypdb" and "from pypdb import * "). On github pypdb.py the function is in comments, but it is not actually in code. Am I missing something? If not, is there another way to do this?
Thanks.

**Comment by williamgilpin** [comment]
thank you, let me check on this. are other functions running, or is just this one giving errors?
if it's just this one function, then it's an API issue I need to fix. Otherwise, I probably broke the pathing in the latest update.

**Comment by pauljmclaughlin** [comment]
Yes, "get_all_info" works, for example. I am not expert enough to know if the code on GitHub is that which is picked up with "import pypdb " within Jupyter Notebooks. But in the python code on GitHub the function looks like a stub .

**Comment by williamgilpin** [comment]
Thank you! It looks like that function has been commented out in the code for the last few releases---my guess is that the function broke when RCSB changed their API last year. Do you know which version of the code you are using, or the date on which the function last worked?

**Comment by williamgilpin** [comment]
the date that the function last worked for you, I mean

**Comment by pauljmclaughlin** [comment]
This is the first time I have tried to use it. In fact, I was alerted to it by a post made three years ago when someone suggested its use, in response to a question. 

(https://bioinformatics.stackexchange.com/questions/8583/retrieve-id-ligand-from-pdb-file)

From this, we can surmise, but not be absolutely sure, that the respondent had used it successfully.

In the interim, I have managed to pull out the ligand names by using the API for PDB. So this is not so crucial to me now. If you are pushed for time, this is not that urgent.

**Comment by williamgilpin** [comment]
I finally got this fixed in the latest GitHub version, can re-open if it is not working

```
>>> get_ligands('100D')['ligandInfo']['ligand']
[{'@structureId': '100D', '@chemicalID': 'SPM', '@type': 'non-polymer',
  '@molecularWeight': 202.34, 'chemicalName': 'SPERMINE',
  'formula': 'C10 H26 N4', 'InChI': 'InChI=1S/C10H26N4/...',
  'InChIKey': 'PFNFFQXMRSDOHW-UHFFFAOYSA-N', 'smiles': 'C(CCNCCCN)CNCCCN'}]
```
